src/main/java/com/github/stefvanschie/inventoryframework/pane/Pane.java

```diff
@@ -501,6 +502,18 @@ public static void load(@NotNull Pane pane, @NotNull Object instance, @NotNull E
         }
     }
 
+    //Code taken from InventoryView#getInventory(rawSlot) to support for 1.12 where method doesn't exist
+    public Inventory getInventory(InventoryView view, int rawSlot) {
+        if(rawSlot == InventoryView.OUTSIDE || rawSlot == -1) {
+            return null;
+        }
+        if(rawSlot < view.getTopInventory().getSize()) {
+            return view.getTopInventory();
+        } else {
+            return view.getBottomInventory();
+        }
+    }
+
```

Comment:
Please remove this method and replace usages of it in this class and extending classes to the one in the `Gui` class (and make that method static), as to avoid duplicate code.

Reply:
done